code/modules/client/preferences.dm

```diff
@@ -522,7 +522,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	return TRUE
 
 /datum/preferences/proc/GetQuirkBalance()
-	var/bal = 0
+	var/bal = 6 //BUBBERSTATION CHANGE: 6 FREE QUIRK POINTS.
```

Comment:
You can't tell me it was this fucking easy and that skyrat just gaslit everyone into thinking it would need a major rework of the quirk system.

Also mandatory "Take it upstream"

Reply:
> mandatory "take it upstream"

Skyrat doesn't have Gifted upstream. It's our quirk. And they don't want people to have free points.

Reply:
@IgiariValkyr  they do

Reply:
wow, this would of been so fucking hard huh? :trollface: